cmd/draino/draino.go

```diff
@@ -63,7 +62,6 @@ func main() {
 		conditions = app.Arg("node-conditions", "Nodes for which any of these conditions are true will be cordoned and drained.").Required().Strings()
 	)
 	kingpin.MustParse(app.Parse(os.Args[1:]))
-	glogWorkaround()
```

Comment:
Can you comment on this - what changed upstream that renders this unnecessary?

Reply:
this workaround `glogWorkaround` was no more compliant with the new set of dependencies. It would simply prevent the program to start by complaining about the not defined parameter `logtostderr`.

`client-go` 1.15 is based on `klogs` a fork of `glog` that has probably solved the problem linked to that workaround.